delivery-status-service/src/main/java/com/tracking_delivery_system/delivery_status_service/model/LocationUpdate.java

```diff
@@ -0,0 +1,21 @@
+package com.tracking_delivery_system.delivery_status_service.model;
+
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+import jakarta.persistence.*;
+import lombok.AllArgsConstructor;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+import java.util.UUID;
+
+@Data
+@NoArgsConstructor
+@AllArgsConstructor
+@Entity
+@Table(name = "location-update")
+@JsonIgnoreProperties(ignoreUnknown = true)
+public class LocationUpdate {
+    @Id
+    private UUID id;
```

Comment:
if it's table so probably you need to have @GeneratedValue(strategy = GenerationType.UUID)